doc/modules/cross_validation.rst

```diff
@@ -665,9 +665,11 @@ Here is a visualization of the cross-validation behavior.
    :scale: 75%
 
 Similar to :class:`KFold`, the test sets from :class:`GroupKFold` will form a
-complete partition of all the data. Unlike :class:`KFold`, :class:`GroupKFold`
-is not randomized at all, whereas :class:`KFold` is randomized when
-``shuffle=True``.
```

Comment:
@zvealey Shouldn't it instead say "when `shuffle` is False" as per code and also
@glemaitre 's [comment](https://github.com/scikit-learn/scikit-learn/pull/28519#discussion_r1526596071) ?

(Maybe too small for a new issue, but too big to ignore)